pom.xml

```diff
@@ -245,6 +245,7 @@
         <version>${maven-surefire-plugin.version}</version>
         <configuration>
           <skip>${skipUTs}</skip>
+          <argLine>-Xms256m -Xmx2g</argLine>
```

Comment:
We must add this line in this PR?

Reply:
it was there when i got OOM in travis...maybe not needed now. let me try removing it